### PR TITLE
Run quick indexes periodically

### PR DESCRIPTION
**Pull request by davidfischer**
- Pretty frequent in dev and half hourly in prod
- Fixes a minor bug with celerybeat locally

### CODE REVIEW

File: config/settings/development.py [hunk 1]
```diff
@@ -35,6 +35,7 @@
 CELERY_TASK_ALWAYS_EAGER = False
 CELERY_BROKER_URL = env("CELERY_BROKER_URL", default=env("REDIS_URL", default=None))
 CELERY_RESULT_BACKEND = CELERY_BROKER_URL
+CELERY_CREATE_MISSING_QUEUES = True
```
Comment (davidfischer) [inline config/settings/development.py:38]: I don't think this is strictly necessary but explicit seems good.

File: docker-compose/local/django/celery/beat/start [hunk 2]
```diff
@@ -5,4 +5,4 @@ set -o nounset
 
 
 rm -f './celerybeat.pid'
-celery -A config.celery_app beat -l INFO -Q celery,analyzer
+celery -A config.celery_app beat -l INFO
```
Comment (davidfischer) [inline docker-compose/local/django/celery/beat/start:8]: This is not necessary on the beat call, only the worker call. We aren't doing this in prod and it was preventing beat from running in dev.

File: config/settings/development.py (hunk 1 above)
Comment (ericholscher) [reply config/settings/development.py:38]: Yea, I think it defaults on, but good to be exlicit.

File: docker-compose/local/django/celery/beat/start (hunk 2 above)
Comment (ericholscher) [reply docker-compose/local/django/celery/beat/start:8]: Odd. I thought it would only listen for 1 by default.

**Review by ericholscher — APPROVED** [review]

**Pull request merged**